test-packages/cspell-eslint-plugin/test-cspell-eslint-plugin-v9/.eslint.config.js

```diff
@@ -22,8 +16,7 @@ const config = [
             '@cspell/spellchecker': ['warn', { checkIdentifiers: true }],
         },
     },
-    ...compat.extends('plugin:prettier/recommended', 'plugin:@typescript-eslint/recommended'),
-    eslintConfigPrettier,
+    ...tsEslint.configs.recommended,
```

Comment:
Maybe this is the issue, since It doesn't look like this was imported the same here